syntax = "proto2";

package LEYc;

message Channel {
  required string c_name = 1;
  optional string c_link = 2;
  
  message Description {
    optional string c_desc_1 = 1;
    optional string c_desc_2 = 2;
  }

  optional Description c_desc = 3;
}

message ChannelList {
  repeated Channel channel = 1;
}
